staging/publishing/rules.yaml

```diff
@@ -1646,3 +1646,15 @@ rules:
       dir: staging/src/k8s.io/controller-manager
     name: release-1.19
     go: 1.14.6
+- destination: mount-utils
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/mount-utils
+    name: master
+  - source:
+      branch: release-1.19
```

Comment:
@nikhita do we need the release-1.19 section since this is not in 1.19?

Reply:
Yikes, my bad. Sorry about that! It was correct before, this should be removed :+1: